mne/gui/tests/test_coreg_gui.py

```diff
@@ -146,22 +153,42 @@ def test_coreg_model():
     finally:
```

Comment:
Isn't this test above redundant? `x._init_plot()` should also be called below in `frame.edit_traits()`